internal/services/analysisservices/analysis_services_server_resource_test.go

```diff
@@ -521,7 +522,10 @@ func (t AnalysisServicesServerResource) suspend(ctx context.Context, clients *cl
 		return err
 	}
 
-	if err := client.SuspendThenPoll(ctx, *id); err != nil {
+	ctx2, cancel := context.WithTimeout(ctx, 15*time.Minute)
```

Comment:
Could we give this a better name 
```suggestion
	timeout, cancel := context.WithTimeout(ctx, 15*time.Minute)
```

Reply:
fixed.